roles/upload-site/tasks/main.yml

```diff
@@ -57,126 +54,17 @@
     - "ev 'acsf_openid_allow_local_user_logins();'"
     - 'ev "_paranoia_remove_risky_permissions();"'
     - "-y dis googleanalytics pingdom_rum"
+    - "-y dis stanford_afs_quota"
     - "sqlq 'truncate table webauth_roles_history'"
     - "sspwmd"
   notify: Clear site cache
 
-# Copy files first so that they exist and Drupal can find them if needed
-- name: Copy files from local to Site Factory
+# Copy public files first so that they exist and Drupal can find them if needed
+- name: Copy public files from local to Site Factory
   shell: "drush -y rsync /tmp/{{ inventory_hostname }}/files/ @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/"
```

Comment:
Probably should add `--exclude-paths=sites/default/files/js_injector/` for https://stanfordits.atlassian.net/browse/SITES-482

Reply:
Updated

Reply:
Changed on this branch.